externals/kyuubi-hive-sql-engine/src/main/scala/org/apache/kyuubi/engine/hive/session/HiveSessionImpl.scala

```diff
@@ -21,16 +21,20 @@ import java.util
 
 import scala.collection.JavaConverters._
 
+import com.google.common.collect.Sets
```

Comment:
be careful on using Guava API in Hive engines, the relocation will be broken once we invoke the Hive method which leaks Guava classes. we already suffered that in thrift and eventually had to create a pre-relocated thrift jar to work around it.